src/gateway/get-started/comprehensive/manage-teams.md

```diff
@@ -123,184 +59,60 @@ http :8001/workspaces \
 
-{% endnavtab %}
-{% endnavtabs %}
-
 ## Create an Admin
 
 Next, create an admin for the SecureWorkspace, granting them permissions to manage only that workspace.
 
-{% navtabs %}
-{% navtab Using Kong Manager %}
-### Invite a New Admin
-
-<div class="alert alert-warning">
-<strong>Note:</strong> If you also use the Admin API, once you've created this admin, you can find it under the <em>/admins</em> endpoint.</div>
-
-1. From the **Teams** > **Admins** tab, click **Invite Admin**.
-2. Enter the new administrator’s **Email** address, **Username**, and **Custom Id**.
-3. Ensure that **Enable RBAC Token** is enabled.
-
-    **Note:** This setting lets the admin use the Admin API as well as Kong Manager. If you don’t want this user to access the Admin API, uncheck this box.
-
-4. Click **Add/Edit Roles**.
-5. In the Workspace Access dialog, select the **SecureWorkspace**.
-6. Select the **workspace-admin** role, which makes this user the workspace administrator for the SecureWorkspace.
-
-    When you are done adding roles, you are redirected back to the Invite Admin dialog.
-
-    <div class="alert alert-warning">
-    <strong>Important:</strong> Before you move on, make sure the <strong>Enable RBAC Token</strong> checkbox is checked. The RBAC token is what allows the new admin to send a token to the Admin API to configure the system programmatically.
-    </div>
-
-7. Click **Invite Admin** to send the invite.
-
-    At this point in the getting started guide, you likely haven’t set up SMTP yet, so no email will be sent. Instead, you’ll later generate a registration link for the new administrator manually.
-
-#### Register the Admin
-
-1. Back on the **Teams** page, click **View** for the administrator you just created.
-2. Click the **Generate registration link** button.
-
-    Using this link, the new administrator can go to a web browser and paste it in to initiate his/her account and create an initial password. Again, normally, this would happen through SMTP, and the user would get this link through an email.
-
-3. Click the **copy icon** to copy the registration link, then save it.
-4. Email or SMS the registration link to the new administrator &mdash; or use it yourself to test the login in the following steps.
-5. Open a different browser or an incognito tab in the current browser so your existing login session is ignored.
-6. Enter the registration link you copied previously into the new browser to log in with the new administrator (secureworkspaceadmin).
-
-    If the registration link has expired, you can generate a new one by logging in with your `kong_admin` administrator and generating a new link.
-
-7. Enter a new password for your new administrator (save this in a secure place) and click on the **Register** button.
-
-    If everything went well, you should see an “Account Setup Success” message.
-{% endnavtab %}
-{% navtab Using the Admin API %}
-
 <div class="alert alert-warning">
```

Comment:
Should this note be in Markdown?

Reply:
it should, but we can comb through it later when we edit these docs.